### PR TITLE
[ON HOLD PENDING DNS SWITCH] Launch code.mil

### PR DESCRIPTION
**Pull request by fulldecent**
Our work in progress branch for the new code.mil website is currently in branch static-site. This pull request represents the initial launch of code.mil by pushing new code.

In the interest in actually launching, we will need to:

- [x] Identify the stakeholders that have authority over code.mil -- @jordangov and @shawoods
- [x] Identify the go-no go decision process that stakeholders have -- @jordangov will review with engineers and decide
- [x] review all content, update as needed
- [x] Launch (switch DNS)
- [x] switch TravisCI deployment config
- [x] merge in new code

Launch is a beautiful thing, but it's not such a big deal. Once you are launched, it will attract more contributors that get you to where you really want to be!

### CODE REVIEW

**Comment by jordangov** [comment]
Thanks for setting this up. 👍

**Comment by fulldecent** [comment]
@jordangov Looks good, I see a bunch of commits just push, thought you wrote them earlier.

I know there's much to do on this project. But may I please ask for help with checkboxes on the top of this PR. Specifically I just wanted to know if stakeholders are identified -- not just you, the technical owner here, but the actual list of people that will sign off on launch. Also if any of those people have any specific requirements that must be met then it would be nice to identify them.

Anything on that list is my top priority so we can launch 1.0. Also if you can add any other hit list or breakers I'll be happy to contribute for sure.

**Comment by jordangov** [comment]
I can understand your desire to have this nailed down, but that's a much more loose process within DDS. There will be a review, but it will mostly be on me to say I'm satisfied. I imagine @shawoods will want a review, and I'll have some other engineer(s) give it a technical review as well.

**Comment by fulldecent** [comment]
Understood. Thank you for that clarification. I have updated the original post with this information.

**Comment by fulldecent** [comment]
Just a note to anyone else following along: deployments on this branch are hitting http://code-dot-mil-website.s3-website-us-east-1.amazonaws.com/

**Comment by jordangov** [comment]
Sorry @fulldecent, but for now we're trying to keep that under wraps. We're only at internal review stage for now.

**Comment by jordangov** [comment]
This PR will be merged into `master`, not the GH pages branch. The deployment will not be to GH pages. We're very close here! Need to resolve some conflicts and we'll be (mostly) set.

**Comment by jordangov** [comment]
@arichiv-usds The `master` branch now requires a mandatory (approved) review before merging a PR. Can you please give this PR another once over and approve?

**DO NOT MERGE** just yet, waiting on the DNS to switch and propagate.

**Comment by jordangov** [comment]
@jgarber623-gov I know you're sick today, but if you can review that would be awesome.

**Comment by fulldecent** [comment]
Hold the phone -- https://github.com/deptofdefense/code.mil/pull/146

**Comment by jordangov** [comment]
:) Thanks @fulldecent. I'll merge those in.

Of course, there will _always_ be more typos. Don't worry, this isn't a deploy-once-ignore-forever project!

**Comment by fulldecent** [comment]
My other last-minute change is #148. I have no more PRs coming today.

Get the bubbly ready.

**Comment by jordangov** [comment]
Commented on that PR, it's going to need to wait until after launch - and after a design review.

**Comment by fulldecent** [comment]
And of course, there is one commit missing before hitting the button.

        branch: static-site

But you get the honors on that one!

**Comment by fulldecent** [comment]
Yes, just saw it's good now. Ready to merge.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+_site/
+.DS_Store
```
Comment (jgarber623-gov) [inline .gitignore]: Recommend using [GitHub's Jekyll `.gitignore` template](https://github.com/github/gitignore/blob/master/Jekyll.gitignore) here.

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+.hidden{
+  display: none;
+}
+
+.usa-hero-callout .usa-button {
+  margin-top: 4rem;
+}
+
+.usa-banner-guidance-ssl {
+  /* This is terrible... but we don't have an SSL cert yet sooooo.... */
```
Comment (jgarber623-gov) [inline assets/css/main.css]: Is this still the case?

File: assets/css/main.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+.tree-node-answer {
+  font-size: 1.1em;
+  font-weight: bold;
+  text-align: center;
+  border-top: 1px solid #999;
+  border-bottom: 1px solid #999;
+  width: 60%;
+  margin: 1em auto;
+}
```
Comment (jgarber623-gov) [inline assets/css/main.css]: Is there a project convention for organizing properties within a CSS rule set? My preference is alphabetized and tools like [sass-lint](https://github.com/sasstools/sass-lint) can help. Not sure how concerned y'all are about this particular matter, though.

File: .gitignore (hunk 1 above)
Comment (fulldecent) [reply .gitignore]: But still keep `.DS_Store`.

Comment (jordangov) [reply .gitignore]: I'm open to any of this, but can wait until after merge/launch

File: assets/css/main.css (hunk 2 above)
Comment (jordangov) [reply assets/css/main.css]: Yes. Sadly. We're working on it soon.

File: assets/css/main.css (hunk 3 above)
Comment (jordangov) [reply assets/css/main.css]: There is not yet, but you can make one. ;)
Honestly, there's so little CSS right now it's hard to think of it.

**Comment by jordangov** [comment]
I think everything that needs to be addressed before merge is done. Waiting on the DNS switch to actually merge this bad boy.

**Comment by jordangov** [comment]
What's this? http://www.code.mil

We're still working out some kinks... like code.mil not redirecting... and I need to update the s3 bucket because DNS. We're still in a holding pattern.

**Comment by jordangov** [comment]
What's this? http://www.code.mil

We're still working out some kinks... like code.mil not redirecting... and I need to update the s3 bucket because DNS. We're still in a holding pattern.

**Pull request merged**